src/node.cc

```diff
@@ -4093,8 +4093,8 @@ int EmitExit(Environment* env) {
   Local<Object> process_object = env->process_object();
   process_object->Set(env->exiting_string(), True(env->isolate()));
 
-  Local<String> exitCode = env->exit_code_string();
-  int code = process_object->Get(exitCode)->Int32Value();
+  Local<String> exit_code = env->exit_code_string();
+  int code = process_object->Get(exit_code)->Int32Value();
```

Comment:
We might as well switch to the non-deprecated versions of `Int32Value()` and `->Get()` here if the PR already touched the code :)